x/emissions/types/params.go

```diff
@@ -48,6 +48,7 @@ func DefaultParams() Params {
 		RegistrationFee:                 cosmosMath.NewInt(6),                      // how much workers and reputers must pay to register per topic
 		DefaultLimit:                    uint64(100),                               // default limit for pagination
 		MaxLimit:                        uint64(1000),                              // max limit for pagination
+		MinEpochLengthRecordLimit:       int64(3),                                  // minimum number of epochs to keep records for a topic
```

Comment:
you need to add this new param to all the other places mentioned in [this document](https://www.notion.so/upshothq/Adding-New-Global-Parameters-Checklist-6216783e3b4f4280a2fda6f3487f0164)